docs/en/guide/configurations.md

```diff
@@ -142,54 +145,237 @@ globalConfig:
+
+```shell
+juicefs mount ... --cache-size=204800 -o writeback_cache,debug
+```
+
+It would translate to CSI `mountOptions` as follows:
+
+```yaml
+mountOptions:
+  # JuiceFS mount options
+  - cache-size=204800
+  # Extra FUSE options
+  - writeback_cache
+  - debug
+```
 
 :::tip
-Starting from v0.24, CSI Driver can customize mount pods and sidecar containers in the [ConfigMap](#configmap), legacy method introduced in this section is not recommended.
+Mount options are different between Community Edition and Cloud Service, see:
+
+- [Community Edition](https://juicefs.com/docs/community/command_reference#mount)
+- [Cloud Service](https://juicefs.com/docs/cloud/reference/commands_reference/#mount)
 :::
 
-Some of the fields that doesn't support CSI Node inheritance, are customized using the following fields in the code block, they can be defined both in storageClass parameters (for dynamic provisioning), and also PVC annotations (static provisioning).
+#### Via ConfigMap
+
+Minimum required version is CSI Driver v0.24.7. Upon modification, application pods need to be re-created for changes to take effect.
+
+Items inside ConfigMap comes with the highest priority, mount options defined in CM will recursively overwrite those defined in PV. So in order to avoid confusion, please migrate all mount options towards ConfigMap and refrain from using PV-level `mountOptions`.
+
+By using `pvcSelector`, you can control mount options for multiple PVCs.
+
+```yaml
+  mountPodPatch:
+    - pvcSelector:
+        matchLabels:
+          # Applies to all PVCs with this label
+          need-update-options: "true"
+      mountOptions:
+        - writeback
+        - cache-size=204800
+```
+
+#### Via PV definition (deprecated) {#static-mount-options}
+
+After modifying the mount options for existing PVs, you need to perform a rolling upgrade or re-create the application pod, so that CSI Driver starts re-create the mount pod for the changes to take effect.
+
+```yaml {8-9}
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: juicefs-pv
+  labels:
+    juicefs-name: ten-pb-fs
+spec:
+  mountOptions:
+    - cache-size=204800
+  ...
+```
+
+#### Via StorageClass definition (deprecated) {#dynamic-mount-options}
+
+Customize mount options in `StorageClass` definition. If you need to use different mount options for different applications, you'll need to create multiple `StorageClass`, each with different mount options.
+
+Due to StorageClass being the template used for creating PVs, **modifying mount options in StorageClass will not affect existing PVs**, if you need to adjust mount options for dynamic provisioning, you'll have to delete existing PVCs, or [directly modify mount options in existing PVs](#static-mount-options).
```

Comment:
```suggestion
Since StorageClass serves as a template for creating PVs, **modifying mount options in StorageClass will not affect existing PVs**. If you need to adjust mount options for dynamic provisioning, you have to delete existing PVCs, or [directly modify mount options in existing PVs](#static-mount-options).
```